pulser-pasqal/pulser_pasqal/pasqal_cloud.py

```diff
@@ -97,6 +127,7 @@ def submit(self, sequence: Sequence, **kwargs: Any) -> RemoteResults:
                     "of the devices currently available through the remote "
                     "connection."
                 )
+            # TODO: Validate the register layout
```

Comment:
I don't think that's necessary : The register layout of the sequence matches with the register layout of the device, or we know the device of the sequence matches with an available device, so this available device has a register layout that can implement the register layout of the sequence. Am I right ?

Reply:
There is nothing in `pulser-core` imposing that a register layout is within `Device.pre_calibrated_layouts`, this is what we need to enforce here.